pages/mobile/page_provider.py

```diff
@@ -7,7 +7,7 @@
 from pages.page import Page
 
 
-class PageProvider(Page):
+class MobilePageProvider(Page):
```

Comment:
You don't really need to rename this here (and in the other places where it is used). All you need to do is alias it in the one place where you are using both `PageProviders`. I will comment on how to do that in that file.